docs/content/reference/dataloaders.md

```diff
@@ -137,7 +137,9 @@ func NewLoaders(conn *sql.DB) *Loaders {
 }
 
 // Middleware injects data loaders into the context
-func Middleware(loaders *Loaders, next http.Handler) http.Handler {
+func Middleware(conn *sql.DB, next http.Handler) http.Handler {
+	// create a request-scoped data loader
+	loader := NewLoaders(conn)
```

Comment:
I don't see where this is used in the below code.

Reply:
https://github.com/99designs/gqlgen/pull/2719/files#diff-31a01bd1c23b054ddad58a6fd10835c9ff3a80d50d32ef18c7d1ab319a758522R175 ?

Reply:
ah @chadxzs is right, it's a typo and should be loader below. I'll open a subsequent fix, thanks